packages/contracts/helpers/contracts-helpers.ts

```diff
@@ -39,27 +45,29 @@ import { getDefenderRelaySigner, usingDefender } from "./defender-utils";
 
 export type MockTokenMap = { [symbol: string]: MintableERC20 };
 
+export const isProdNetwork = (network: string): boolean => {
+  return (
+    network === "sepolia" ||
+    network === "optimism" ||
+    network === "base" ||
+    network === "arbitrum"
+  );
+};
+
 export const registerContractInJsonDb = async (
   contractId: string,
   contractInstance: Contract
 ) => {
   const currentNetwork = DRE.network.name;
-  const FORK = process.env.FORK;
-  if (
-    FORK ||
-    (currentNetwork !== "hardhat" && !currentNetwork.includes("coverage"))
-  ) {
-    // console.log(`*** ${contractId} ***\n`);
-    // console.log(`Network: ${currentNetwork}`);
-    // console.log(`tx: ${contractInstance.deployTransaction.hash}`);
-    // console.log(`contract address: ${contractInstance.address}`);
-    // console.log(`deployer address: ${contractInstance.deployTransaction.from}`);
-    // console.log(`gas price: ${contractInstance.deployTransaction.gasPrice}`);
-    // console.log(`gas used: ${contractInstance.deployTransaction.gasLimit}`);
-    // console.log(`\n******`);
-    // console.log();
-  }
 
+  if (isProdNetwork(currentNetwork)) {
```

Comment:
so is the workflow to write separately to prod db and never have to copy from deployed-contracts again? If so, there's a lot of other places where we write to regular db. Like all the current deployment scripts under tasks will write to the regular db

Reply:
yes, we don't have to manually copy. 

All deployment scripts will write to `deployed-contracts.json`. Only prod networks will write to the prod json. I don't understand what the issue is.